src/css/modules/_panel.scss

```diff
@@ -211,6 +211,55 @@
   }
 }
 
+.panel__simplepage {
+
+  p {
+    max-width: 45em;
+    margin:0 auto;
+    font-size:1.2em;
+    padding-left:2.5em;
+    padding-right:2.5em;
+    span {
+      font-family: Archer, Archer SSm A, Archer SSm B;
```

Comment:
I think that this archer should be a variable and include a backup. Something like: 

```
$font-special: Archer, Archer SSm A, Archer SSm B, serif;
```